core/core.go

```diff
@@ -271,10 +271,7 @@ func (c *Core) WriteBlock(block *types.Block) {
 func (c *Core) Append(header *types.Header, domPendingHeader *types.Header, domTerminus common.Hash, domOrigin bool, newInboundEtxs types.Transactions) (types.Transactions, bool, error) {
 	newPendingEtxs, subReorg, err := c.sl.Append(header, domPendingHeader, domTerminus, domOrigin, newInboundEtxs)
 	if err != nil {
-		if err.Error() == ErrBodyNotFound.Error() {
-			c.sl.missingBodyFeed.Send(header)
```

Comment:
You just removed the invocation of `missingBodyFeed`. Still need to remove the feed itself